client/src/components/backend/Drawer/Wrapper.js

```diff
@@ -11,11 +11,13 @@ export default class DrawerWrapper extends PureComponent {
   static propTypes = {
     closeUrl: PropTypes.string,
     closeCallback: PropTypes.func,
-    lockScroll: PropTypes.bool
+    lockScroll: PropTypes.bool,
+    style: PropTypes.string,
```

Comment:
This should be made required, and you should use the oneOf react proptype to specify that it must be backend or frontend.